template_workspace.yaml

```diff
@@ -12,4 +12,4 @@ repositories:
   sdformat_urdf:
     type: git
     url: https://github.com/ros/sdformat_urdf.git
-    version: ros2
+    version: rolling
```

Comment:
Thanks for updating the branch but making it default to `rolling` won't allow choosing any other distribution. Also some combinations have this package's binaries available so for those we don't need to build by source.